user/lookup.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2011 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package user
```

Comment:
Why do we need a new package for this ? The name is not good, pkg/sftp/user doesn't explain what this package provides. 

Can it be merged into the sftp package please.